Front end react bootstrap navbar

knt-zhang opened the pull request:
For issues#12, 13, 14.

knt-zhang committed:
implement navbar

knt-zhang committed:
Set up main page

Co-Authored-By: Rohan Tohaan <32985555+rtohaan@users.noreply.github.com>

knt-zhang committed:
Merge branch 'master' into Front-end-react-bootstrap-Navbar

knt-zhang committed:
created pages folder

Close Issue#12, 13, 14

Co-Authored-By: Rohan Tohaan <32985555+rtohaan@users.noreply.github.com>

Suchwinder reviewed (changes requested):
For the navbar please make it a separate component that can be imported to the other pages, please create a separate folder within components. 
e.g: components/navbar
In there please create navbar.jsx, and this file should have the view/functionality of the navbar. From there we can import this component directly to each of our pages.

knt-zhang commented:
@Suchwinder I'm having an issue when importing the Navbar, "You attempted to import /components/navbar which falls outside of the project src/ directory."

Suchwinder committed:
Modularize Nav Bar

Updated front end to have a nav bar component to be imported into all our other pages, in addition code refactored file structure

Suchwinder commented:
> @Suchwinder I'm having an issue when importing the Navbar, "You attempted to import /components/navbar which falls outside of the project src/ directory."

No problem, please refer to updated code base.
